bf2fc6cc711aee1a0c2a/kas-fleet-manager
# bug: replace getSSoProviders with getSsoProvider

## Description
The function is named getSsoProvider [in service_accounts.go line 176](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/kafka/internal/handlers/service_accounts.go#L176), and called throughout serviceaccounts_test.go, and route_loader.go

`make  openapi/generate` will generate api_security.go and openapi.yaml, referencing the non existent function getssoProviders from [kas-fleet-manager.yaml](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/openapi/kas-fleet-manager.yaml#L680)

This renaming will resolve the bug.

## Verification Steps
1.  `make  openapi/generate`  before the change, and observe the conflict.
2.  `make  openapi/generate`  after the change, and observe.


## Checklist (Definition of Done)
